documents.go

```diff
@@ -47,3 +60,68 @@ func (cl *DocumentsClientGetAll) Parent(id DocumentID) *DocumentsClientGetAll {
 func (cl *DocumentsClientGetAll) Do(ctx context.Context, fn func(*Document, error) bool) error {
 	return nil
 }
+
+// Create returns a client for creating a single document in the specified collection.
+// collectionID: The ID of the collection to which the document will be added.
+// title: The title of the document (optional, set to empty string to omit).
+// text: The content text of the document.
+// publish: A boolean indicating whether the document should be published.
+func (cl *DocumentsClient) Create(collectionID CollectionID, title string, text string, publish bool, parentDocumentId ParentDocumentID, templateId TemplateID, template bool) *DocumentsClientCreate {
+	return newDocumentsClientCreate(cl.sl, collectionID, title, text, publish, parentDocumentId, templateId, template)
+}
+
+// DocumentsClientCreate is a client for creating a single document.
+type DocumentsClientCreate struct {
+	sl *sling.Sling
+}
+
+// newDocumentsClientCreate creates a new DocumentsClientCreate instance to create a single document in the specified collection.
```

Comment:
Do we really need this extensive documentation for private functions? 🤔 
Espeically because some of them are Self-explanation.
Like `collectionId` or `title` or `text` and so one 😁 

I would rather suggest to add the link to the outline API so in cae someone has to lookup something how the API works, we can press on it.